### PR TITLE
Fix/ui fixes

### LINKED ISSUE #535: Small UI issues
**Describe the bug**
Four small UI bugs:
1. Change 'All Districts' in the District filter value container to 'ALL DISTRICTS' to align on capitalization across all of the filters. Maybe create a utility for this that all filter strings get run through to ensure consistency moving forward.
2. Update the toggles bar to make it more responsive. It looks fine until iPad, at which point things get out of line. 
--Ensure that the Supervision Level and Supervision Type filter labels do not wrap. If the font size becomes too small to ensure the no-wrap, then maybe we can just set the label height such that the filters all still align horizontally.
--Ensure that the value container text does not wrap. This is mostly an issue on the District Filter. We may need to insert an ellipsis "..." at the end of the text to indicate that some of the text is not visible like we do on the other filters.
3. On smaller screens, the ‘Risk Level’ header for the sub-charts becomes oddly centered, instead of left-justified as the rest are. Ensure that Risk Level header is left centered at all screen sizes.
4. Y-axis label on “Number of admitted individuals” chart gets cut off when the time period is set to “3 years” on the `RevocationsOverTime` chart.
5. On top “Admissions to SCI from parole” chart, on data point hover, “Revocations” should be changed to “Admissions”
image

**To Reproduce**

**Expected behavior**

**Screenshots**
Screenshots revealing sensitive information _must not_ be shared. If a screenshot is helpful in diagnosing and resolving the bug, please send this directly to a contact on the Recidiviz team.

**Desktop (please complete the following information):**
 - OS [e.g. Windows 10]: 
 - Browser [e.g. chrome, edge]: 
 - Version [e.g. 22]: 

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]: 
 - OS: [e.g. iOS8.1]: 
 - Browser [e.g. stock browser, safari]: 
 - Version [e.g. 22]: 

**Additional context**

### PR DESCRIPTION
**Pull request by Wmaileq**
## Description of the change

1. Values in select now uppercased
2. Fixed 768-1024 top bar filters UI
3. Fixed text-align of sub charts headings
4. Increased chart height
5. Added translations for revocations\admissions

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #535 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by jovergaag** [comment]
@jessex I'm hoping you can take a look at my two commits at the end here. I reviewed all of the commits from @Wmaileq which looked great. I did end up to checking with Sonia on desired behavior and look of the districts filter on tablet. After trying a few options, she asked that we change the value labels that are being used so that 'ALL DISTRICTS' and '3 ITEMS SELECTED' wasn't getting cut off, while still giving the user access to the cursor for searching districts. We now display 'ALL' (which matches the other filters) and '3 ITEMS'.

I would love to get this merged today if you have a chance to take a peek. Thanks!

File: src/components/charts/new_revocations/ToggleBar/DistrictFilter.js [hunk 1]
```diff
@@ -34,11 +34,7 @@ import {
   getUserAppMetadata,
 } from "../../../../utils/authentication/user";
 
-const allDistrictsOption = { label: "All districts", value: "All" };
-const allRegionDistrictsOption = {
-  label: "All region districts",
```
Comment (jessex) [inline src/components/charts/new_revocations/ToggleBar/DistrictFilter.js:39]: Was this struck based on feedback from design?

**Review by jessex — APPROVED** [review]
Just one question.

File: src/components/charts/new_revocations/ToggleBar/DistrictFilter.js (hunk 1 above)
Comment (jovergaag) [reply src/components/charts/new_revocations/ToggleBar/DistrictFilter.js:39]: Yes. We have a suggestion to change the default value strings on both lantern dashboards and on ND. It accomplishes two things, 1) is more consistent with labels on the other filters and 2) looks nicer on tablet.

**Pull request merged**